### PR TITLE
Update code in Yggdrasil/Furnace

### PR DESCRIPTION
**Pull request by CloudeaSoft**
Add enable check to this item.

### CODE REVIEW

**Comment by CloudeaSoft** [comment]
### Incomplete Tasks:
- [ ] Images
   - [ ] Heat Emblem
   - [ ] Melter Gear
   - [ ] Thermal Conductor
      - [ ] Normal
      - [ ] Cooldown Buff
   - [ ] Hot Air-balloon
      - [ ] Normal
      - [ ] _Balloon
   - [ ] Thermostat Rod
      - [ ] Normal
      - [ ] Bobber
- [ ] Code (Ensure that the buffs provided by the following accessories can be adjusted for Expert/Master Mode without being increased)
   - [ ] Heat Emblem
   - [ ] Melter Gear
   - [ ] Thermal Conductor

File: Sources/Modules/Yggdrasil/Everglow.Yggdrasil.csproj [hunk 1]
```diff
@@ -5,4 +5,7 @@
   <ItemGroup>
 		<Solaestas-ResourceFile Include="YggdrasilTown\**\*.json" />
   </ItemGroup>
+  <ItemGroup>
+    <None Remove="Furnace\Buffs\MelterGearCooldown.png" />
```
Comment (Solaestas) [inline Sources/Modules/Yggdrasil/Everglow.Yggdrasil.csproj:9]: These are accidental change, which should be deleted

File: Sources/Modules/Yggdrasil/Furnace/Items/Accessories/HotAirBalloon.cs [hunk 2]
```diff
@@ -43,9 +43,21 @@ public override void PreUpdate()
...
 		{
-			if (Main.rand.NextBool(6))
+			if (!HotAirBalloonEnable)
 			{
-				Dust.NewDust(Player.position, Player.width, Player.height, DustID.Firefly);
+				return;
+			}
+
+			// Check if the player is moving
+			if (Player.velocity == Vector2.Zero)
```
Comment (Solaestas) [inline Sources/Modules/Yggdrasil/Furnace/Items/Accessories/HotAirBalloon.cs:52]: Have you ever considered a player in the air?
Terraria treat player in the air by giving them a slight vertical speed (0.00001)

File: Sources/Modules/Yggdrasil/Everglow.Yggdrasil.csproj (hunk 1 above)
Comment (CloudeaSoft) [reply Sources/Modules/Yggdrasil/Everglow.Yggdrasil.csproj]: fixed

File: Sources/Modules/Yggdrasil/Furnace/Items/Accessories/HotAirBalloon.cs (hunk 2 above)
Comment (CloudeaSoft) [reply Sources/Modules/Yggdrasil/Furnace/Items/Accessories/HotAirBalloon.cs]: fixed

**Review by Solaestas — APPROVED** [review]

**Pull request merged**